### PR TITLE
Somewhat loosen the types for the standard binding handlers.

### PR DESCRIPTION
**Pull request by maskmaster**
Allow observable variables to be bound to all standard binding handlers. Also adjust the allowed types for the css and style bindings.

### CODE REVIEW

**Comment by tscpp** [comment]
If valueaccessor unwrap observables, approved. If not, not approved.

**Comment by maskmaster** [comment]
Valueaccessor does not unwrap. I think it is the responsibility for each binding handler to do if they allow observables to be bound. The normal behaviour in a binding handler is to unwrap the result of the valueaccessor-call.
See example from knockouts documentation on custom binding handlers
```javascript
ko.bindingHandlers.slideVisible = {
    update: function(element, valueAccessor, allBindings) {
        // First get the latest data that we're bound to
        var value = valueAccessor();
 
        // Next, whether or not the supplied model property is observable, get its current value
        var valueUnwrapped = ko.unwrap(value);
```
This means that custom bindinghandlers that does not accept observables cannot use the `BindingContextIdentityTransform` that we use.

**Comment by tscpp** [comment]
I think it is the responsibility of each binding handler to use `ko.MaybeObservable` on the type. Else, the binding handler won't accept observables.

When using an observable, the observable should be called and not sent into the binding handler as an observable, if not an observable, is wanted.

File: lib/context.d.ts [hunk 1]
```diff
@@ -123,9 +123,9 @@ export interface StandardBindingContextTransforms {
 	hidden: BindingContextIdentityTransform<boolean>
 	html: BindingContextIdentityTransform<string>
 	class: BindingContextIdentityTransform<string>
-	css: BindingContextIdentityTransform<string | Record<string, boolean>>
-	style: BindingContextIdentityTransform<Record<string, string>>
-	attr: BindingContextIdentityTransform<Record<string, string>>
+	css: BindingContextIdentityTransform<string | Record<string, MaybeReadonlyObservable<boolean>>>
+	style: BindingContextIdentityTransform<Record<string, MaybeReadonlyObservable<string | number>>>
```
Comment (tscpp) [inline lib/context.d.ts:127]: Using numbers as the value in the 'style' binding handler is not supported.

```suggestion
	style: BindingContextIdentityTransform<Record<string, MaybeReadonlyObservable<string>>>
```

**Review by tscpp — CHANGES_REQUESTED** [review]

**Review by tscpp — CHANGES_REQUESTED** [review]
Add changes to CHANGELOG.md, if any noticeably changed, which may not the case here. Let me know when you are finished.

**Comment by tscpp** [comment]
Oops

**Pull request closed without merging**